public/sass/_variables.generated.scss

```diff
@@ -112,7 +112,7 @@ $font-size-md: 14px !default;
 $font-size-sm: 12px !default;
 $font-size-xs: 10px !default;
 
-$line-height-base: 1.5 !default;
+$line-height-base: 1.5714 !default;
```

Comment:
think this change wants removing otherwise it just gets unset when running locally as the corresponding changes in the theme aren't there